File: JavaScript/u10/Student.js
export default class Student {
    constructor(name,age,sex,id) {
        this.name = name;
        this.age = age;
        this.sex = sex;
        this.id = id;
    }
    toString() {
        return `姓名：${this.name} 年龄：${this.age}性别：${this.sex}学号：${this.id}`;
    }
}

File: Java/考试/Student.java
public class Student extends Person {

    public Student (String name, String age) {
        super.setName(name);
        super.setAge(age);
    }
	@Override
	public void Study() {
		System.out.println("姓名："+super.getName()+" 年龄："+super.getAge()+" 学习Java"); 
	}
}
class Studenttest{
    public static void main(String[] args) {
        Person p=new Student("帅哥", "20");
        p.Study();
    }
}